lib/ops/opsclient/opsclient.go

```diff
@@ -443,16 +443,20 @@ func (c *Client) CheckSiteStatus(ctx context.Context, key ops.SiteKey) error {
 	return trace.Wrap(err)
 }
 
-func (c *Client) GetSiteOperations(siteKey ops.SiteKey) (ops.SiteOperations, error) {
+func (c *Client) GetSiteOperations(siteKey ops.SiteKey, f ops.OperationsFilter) (ops.SiteOperations, error) {
 	out, err := c.Get(context.TODO(), c.Endpoint("accounts", siteKey.AccountID, "sites", siteKey.SiteDomain, "operations", "common"),
-		url.Values{})
+		f.URLValues())
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 	var ops ops.SiteOperations
 	if err := json.Unmarshal(out.Bytes(), &ops); err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	// Servers that haven't been upgraded may not have filtered the results.
+	ops = f.Filter(ops)
```

Comment:
I would really handle this in another API since otherwise, we're always handling it twice when communicating with a remote operator.

Reply:
Why does that matter?